src/platform/input/game-pads.js

```diff
@@ -147,6 +147,7 @@ const PRODUCT_CODES = {
 };
 
 let deadZone = 0.25;
+const previous = [];
```

Comment:
I am not a big fan of these globals as they conflict with each when running multiple `pc.Application`'s with two `pc.GamePads`

IMO it should be `GamaPads#_previous = [];`

Reply:
It's changed, thanks!